metadata/install-qa-check.d/60python-site

```diff
@@ -142,6 +143,16 @@ python_site_check() {
 				')' -a '!' -name "*${ext_suffix}" -print0
 		)
 
+		# check for namespace packages
+		while IFS= read -d $'\0' -r f; do
```

Comment:
It looks like most of the eclasses use this to specify NUL as a delimiter:
```suggestion
		while IFS= read -d '' -r f; do
```

Reply:
Yes, but `$'\0'` is more explicit (the short form has confused people in the past), and it's used elsewhere in the file.